src/components/pages/Organization.tsx

```diff
@@ -72,6 +72,7 @@ const Organization = () => {
           )}
         </Grid>
         <Grid item xs>
+          {organization?.contactInformation}
```

Comment:
Why is this added?

Reply:
I had made the create project action use the `can_edit_project_details` permission, in which case we should totally hide this paper if there's no contact info and we can't create a project. So I think this was initially a condition to check for that. I thought better of using that permission, so I removed the permission filter, but must have not completely removed the code that would hide the paper, thus this line. I'll remove it